content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/clientlibs/editor/js/image.js

```diff
@@ -604,49 +612,12 @@
             mutations.forEach(function(mutation) {
                 if (mutation.type === "attributes") {
                     var isAltCheckboxChecked = $(altCheckboxSelector).attr("checked");
-                    var alertIcon = $(altInputAlertIconSelector);
-                    var assetTab = $(assetTabSelector);
-                    var assetTabAlertIcon = $(assetTabAlertIconSelector);
-                    if (mutation.attributeName === "data-seeded-value") {
-                        if (isAltCheckboxChecked) {
-                            if ($(altInputSelector).val()) {
-                                if (alertIcon.length) {
```

Comment:
5 nested "if"s - what could go wrong?
now it's much more readable 👍

Reply:
Yes, this things should be avoided